plugins/outputs/cloudwatch/cloudwatch_test.go

```diff
@@ -59,6 +59,108 @@ func TestBuildDimensions(t *testing.T) {
 	}
 }
 
+// Test that global dimensions are added correctly
+func TestGlobalDimensions(t *testing.T) {
+	assert := assert.New(t)
```

Comment:
I don't think we need this. I'm assuming this is copied from the other tests but I don't think it's necessary

Reply:
It's not strictly needed but it is the pattern used in all of the surrounding tests and means that each assertion is _very slightly_ more terse:

```go
assert := assert.New(t)
assert.Equal(MaxDimensions, len(dimensions), "Number of dimensions should be less than MaxDimensions")
```

vs.

```go
// Need to pass in `t` as the first argument to `assert.Equal()`
assert.Equal(t, MaxDimensions, len(dimensions), "Number of dimensions should be less than MaxDimensions")
```

Reply:
Huh I never noticed that. At any rate, I'm fine leaving it.